crates/figma_import/src/serialized_document.rs

```diff
@@ -52,3 +77,105 @@ pub struct ServerFigmaDoc {
     pub project_files: Vec<FigmaDocInfo>,
     pub errors: Vec<String>,
 }
+
+/// A helper method to load a serialized design doc from figma.
+pub fn load_design_doc<P>(
+    load_path: P,
+) -> Result<(SerializedDesignDocHeader, SerializedDesignDoc), Error>
+where
+    P: AsRef<Path>,
+{
+    let mut document_file = File::open(&load_path)?;
+
+    let mut buf: Vec<u8> = vec![];
+    let _bytes = document_file.read_to_end(&mut buf)?;
+
+    let header: SerializedDesignDocHeader = bincode::deserialize(buf.as_slice())?;
+
+    // Ensure the version of the document matches this version of automotive design compose.
+    if header.version != SerializedDesignDocHeader::current().version {
+        return Err(Error::FigmaError(format!(
```

Comment:
I'd rather say that this is a DesignCompose doc error, since it's a problem with the "DCF" rather than the original Figma doc.

Reply:
I've moved that to a more generic `DocumentLoadError` with a descriptive string so we can report the mismatch in versions in the string.